app/controllers/items_controller.rb

```diff
@@ -21,8 +22,10 @@ def new
 
   def create
     @merchant = Merchant.find(params[:merchant_id])
-    item = @merchant.items.create(item_params)
-    if item.save
+    item = @merchant.items.new(item_params)
+    if (item.loaded_image? || item.image == "") && item.save
+      # item.image = "https://robohash.org/Y9J.png?set=set4" if item.image == ""
```

Comment:
Is this the bug you mentioned?

Reply:
I can't get it to give me a placeholder image.